app/src/main/java/com/example/fittracker/mvp/views/SignUpView.java

```diff
@@ -0,0 +1,78 @@
+package com.example.fittracker.mvp.views;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.example.fittracker.R;
+import com.example.fittracker.User;
+import com.example.fittracker.mvp.contracts.SignUpContract;
+
+import java.lang.ref.WeakReference;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class SignUpView implements SignUpContract.View {
+    private WeakReference<Activity> activity;
+    @BindView(R.id.signup_activity_text_email) EditText mail;
+    @BindView(R.id.signup_activity_text_password) EditText password;
+    @BindView(R.id.signup_activity_text_pass_rep) EditText passwordRep;
+    @BindView(R.id.signup_activity_text_nombre) EditText name;
+    @BindView(R.id.signup_activity_text_apellido) EditText surname;
+
+    public SignUpView(Activity activity) {
+        this.activity = new WeakReference<>(activity);
+        ButterKnife.bind(this, activity);
+    }
+
+    @Override
+    public String getEmail() {
+        return mail.getText().toString();
+    }
+
+    @Override
+    public String getPassword() {
+        return password.getText().toString();
+    }
+
+    @Override
+    public String getName() {
+        return name.getText().toString();
+    }
+
+    @Override
+    public void userAlreadyExists(User user) {
```

Comment:
if you don't use the user, please remove the parameter

Reply:
Done